cli/tests/test_support.bash

```diff
@@ -216,7 +216,9 @@ wait_for_watchdogs() {
 common_test_teardown() {
   # Delete test tmpdir unless the user requests to preserve them.
   # XXX: We do not attempt to delete envs here.
-  if [[ -z "${FLOX_TEST_KEEP_TMP:-}" ]]; then rm -rf "$BATS_TEST_TMPDIR"; fi
```

Comment:
(non-blocking observation) I was surprised that `rm -rf` could complain about a non-empty directory but I guess the internal implementation is equivalent to:
1. `rm flox-data/*`
2. `flox-data/env-registry.json` gets recreated
3. `rmdir flox-data` fails

Reply:
yeah i'm surprised how often 2. sneaks into 1 and 3, wonder if github runners ahve particularly slow file systems or something alike